src/diffusers/models/attention_processor.py

```diff
@@ -109,15 +109,19 @@ def __init__(
         residual_connection: bool = False,
         _from_deprecated_attn_block: bool = False,
         processor: Optional["AttnProcessor"] = None,
+        scale_mask_factor: float = 1.0,
+        out_dim: int = None,
```

Comment:
Is `out_dim` different from `query_dim` here?

Reply:
the only difference is the `to_out`layer here - Kandinsky attention output does not change dimension from `inner_dim`, while our attention class will project the output to `query_dim`. I added an `out_dim` for this purpose, but I think we can certainly add a different config to do this if it makes more sense! 

        self.to_out.append(nn.Linear(out_channels, out_channels, bias=False))`

https://github.com/huggingface/diffusers/blob/d1b2a1a95710964ff84b2c2332cd2ad93734561e/src/diffusers/models/unet_kandi3.py#L453

Reply:
That works! Makes sense